tests/scripts/add-local-ip-auth-networks.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+source scripts/vars.env
+
+CURRENT_AUTH_NETWORK=$(gcloud container clusters describe ${GKE_CLUSTER_NAME} --zone=${GKE_CLUSTER_ZONE} \
+                        --format="value(masterAuthorizedNetworksConfig.cidrBlocks[0])" | sed 's/cidrBlock=//')
+
+gcloud container clusters update ${GKE_CLUSTER_NAME} --zone=${GKE_CLUSTER_ZONE} --enable-master-authorized-networks --master-authorized-networks=${SOURCE_IP_RANGE},${CURRENT_AUTH_NETWORK}
```

Comment:
It would be nice if we could do this conditionally as part of the GKE cluster creation, assuming that we can check an env var to determine if we're in a pipeline or not. I just have a feeling that devs will forget this step sometimes.

Reply:
I'll leave it as a separate target as well, because local IPs can change and this script may have to be ran more than once, but I'll also add it as part of the cluster creation